scripts/update_vertexai_responses.sh

```diff
@@ -0,0 +1,17 @@
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cd "$(dirname "$0")/../FirebaseVertexAI/Tests/Unit" &&
+rm -rf vertexai-sdk-test-data &&
```

Comment:
nit: See https://google.github.io/styleguide/shellguide.html#pipelines for guidance on formatting the chain of commands with `&&` across multiple lines. Or, my personal preference, I like using `|| exit` instead of chaining like:
```
cd "$(dirname "$0")/../FirebaseVertexAI/Tests/Unit" || exit
rm -rf vertexai-sdk-test-data || exit
...
```
I learned about that approach from https://www.shellcheck.net/wiki/SC2164.

Reply:
Very helpful links, thank you!